codegen/generator/nodejs/templates/src/testdata/type_test_scalar_want.ts

```diff
@@ -1,6 +1,5 @@
 
-
-  /**
+/**
```

Comment:
This is not good, it should be indented 1 time (2 spaces).

Reply:
Fixed in https://github.com/dagger/dagger/pull/4367/commits/516b085d5e82abfa2961d86e42104adbf679134a